thriftpy2/_compat.py

```diff
@@ -11,11 +11,9 @@
 
 import platform
 import sys
-from urllib.parse import urlparse
-from urllib.request import urlopen
+from urllib.parse import urlparse  # noqa
+from urllib.request import urlopen  # noqa
```

Comment:
If no codes are using these modules, I think we should remove them.

```suggestion
```

Reply:
https://github.com/Thriftpy/thriftpy2/blob/61ef458059c63fb1fd0181780123de1f36462825/thriftpy2/parser/parser.py#L17 it will import urlparse  from _compat,  I think we should import from urllib directly here and maybe should remove it in another pr